preferences/legacy/metadata/metadata.yaml

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: instancetype.kubevirt.io/v1beta1
+kind: VirtualMachinePreference
+metadata:
+  name: metadata
+  annotations:
+    tags: "hidden,kubevirt,legacy"
+    iconClass: "icon-legacy"
```

Comment:
Does this class exist?

Reply:
No, and I guess there is no much point in adding a specific icon for this. Let's use just `icon-linux`.